opencti-platform/opencti-front/src/private/components/threats/threat_actors_individual/ThreatActorIndividual.tsx

```diff
@@ -155,61 +157,99 @@ const ThreatActorIndividualComponent = ({
     threatActorIndividualFragment,
     data,
   );
+  const { overviewLayoutCustomization } = useAuth();
+  const threatActorIndividualOverviewLayoutCustomization = overviewLayoutCustomization?.get(threatActorIndividual.entity_type) ?? defaultConfiguration;
   return (
     <>
       <Grid
         container={true}
         spacing={3}
         classes={{ container: classes.gridContainer }}
       >
-        <Grid item={true} xs={6} style={{ paddingTop: 10 }}>
-          <ThreatActorIndividualDetails
-            threatActorIndividualData={threatActorIndividual}
-          />
-        </Grid>
-        <Grid item={true} xs={6} style={{ paddingTop: 10 }}>
-          <StixDomainObjectOverview stixDomainObject={threatActorIndividual} />
-        </Grid>
-        {hasDemographicsOrBiographics(threatActorIndividual) && (
-          <>
-            <Grid item={true} xs={6} style={{ marginTop: 30 }}>
-              <ThreatActorIndividualDemographics
-                threatActorIndividual={threatActorIndividual}
-              />
-            </Grid>
-            <Grid item={true} xs={6} style={{ marginTop: 30 }}>
-              <ThreatActorIndividualBiographics
-                threatActorIndividual={threatActorIndividual}
-              />
-            </Grid>
-          </>
-        )}
-        <Grid item={true} xs={6} style={{ marginTop: 30 }}>
-          <SimpleStixObjectOrStixRelationshipStixCoreRelationships
-            stixObjectOrStixRelationshipId={threatActorIndividual.id}
-            stixObjectOrStixRelationshipLink={`/dashboard/threats/threat_actors_individual/${threatActorIndividual.id}/knowledge`}
-          />
-        </Grid>
-        <Grid item={true} xs={6} style={{ marginTop: 30 }}>
-          <StixCoreObjectOrStixRelationshipLastContainers
-            stixCoreObjectOrStixRelationshipId={threatActorIndividual.id}
-          />
-        </Grid>
-        <Grid item={true} xs={6} style={{ marginTop: 30 }}>
-          <StixCoreObjectExternalReferences
-            stixCoreObjectId={threatActorIndividual.id}
-          />
-        </Grid>
-        <Grid item={true} xs={6} style={{ marginTop: 30 }}>
-          <StixCoreObjectLatestHistory
-            stixCoreObjectId={threatActorIndividual.id}
-          />
-        </Grid>
+        {
+          Array.from(threatActorIndividualOverviewLayoutCustomization.entries()).map(([key, { width }]) => {
+            switch (key) {
+              case 'details':
+                return (
+                  <Grid key={'details'} item={true} xs={width} style={{ paddingTop: 10 }}>
```

Comment:
Grid should be outside the switch

Reply:
This will be done after the merge of the Grid style refactoring.